patches/tModLoader/Terraria/Localization/Content/fr-FR/tModLoader.json

```diff
@@ -375,77 +375,77 @@
+		// "ModExceptionHint": "\n\nConseil: C'est probablement la faute d'un mod. Désactivez les mods un par un et vérifiez si le problème persiste",
+		// "OneDriveHint": "\n\nConseil: Essayez d'installer/activer OneDrive. Faites un clic droit sur votre dossier de documents et activez \"Toujours enregistrer sur cet appareil\"",
+		// "AntivirusHint": "\n\nConseil: Si vous avez un antivirus, essayez d’ajouter Terraria à sa liste blanche/d’exclusion",
+		// "TypeInitializationHint": "\n\nConseil: Redémarrez le jeu. Si ce problème persiste, essayez de désinstaller complètement Terraria, puis de réinstaller via Steam, puis de réinstaller tModLoader"
 	},
 	"CommonItemTooltip": {
 		"PercentIncreasedDamage": "+{0} % de dégâts"
 	},
 	"RecipeConditions": {
-		// "NearWater": "Needs water",
-		// "NearLava": "Needs lava",
-		// "NearHoney": "Needs honey",
-		// "TimeDay": "Needs day",
-		// "TimeNight": "Needs night",
-		// "InDungeon": "Needs to be in Dungeon Biome",
-		// "InCorrupt": "Needs to be in Corruption Biome",
-		// "InHallow": "Needs to be in Hallow Biome",
-		// "InMeteor": "Needs to be in Meteorite Biome",
-		// "InJungle": "Needs to be in Jungle Biome",
-		// "InSnow": "Needs to be in Snow Biome",
-		// "InCrimson": "Needs to be in Crimson Biome",
-		// "InWaterCandle": "Needs to be near Water Candle",
-		// "InPeaceCandle": "Needs to be near Peace Candle",
-		// "InTowerSolar": "Needs to be near Solar Tower",
-		// "InTowerVortex": "Needs to be near Vortex Tower",
-		// "InTowerNebula": "Needs to be near Nebula Tower",
-		// "InTowerStardust": "Needs to be near Stardust Tower",
-		// "InDesert": "Needs to be in Desert Biome",
-		// "InGlowshroom": "Needs to be in Mushroom Biome",
-		// "InUndergroundDesert": "Needs to be in Underground Desert",
-		// "InSkyHeight": "Needs to be in Sky",
-		// "InOverworldHeight": "Needs to be in Surface Layer",
-		// "InDirtLayerHeight": "Needs to be in Underground Layer",
-		// "InRockLayerHeight": "Needs to be in Cavern Layer",
-		// "InUnderworldHeight": "Needs to be in Underworld",
-		// "InBeach": "Needs to be in Ocean Biome",
-		// "InRain": "Needs to be during rain",
-		// "InSandstorm": "Needs to be during Sandstorm",
-		// "InOldOneArmy": "Needs to be during Old One's Army",
-		// "InGranite": "Needs to be in Granite Biome",
-		// "InMarble": "Needs to be in Marble Biome",
-		// "InHive": "Needs to be in Hive",
-		// "InGemCave": "Needs to be in Gem Cave",
-		// "InLihzardTemple": "Needs to be in Lihzard Temple",
-		// "InGraveyardBiome": "Needs to be in Graveyard Biome"
+		// "NearWater": "Besoins d’eau",
+		// "NearLava": "Besoins de lave",
+		// "NearHoney": "Besoins de miel",
+		// "TimeDay": "Besoins de jour",
+		// "TimeNight": "Besoin de nuit",
+		// "InDungeon": "Doit être dans le Biome du Donjon",
+		// "InCorrupt": "Doit être dans le biome de la Corruption",
+		// "InHallow": "Doit être dans le biome de la Sacré",
```

Comment:
Why is "biome" sometimes with and sometimes without an uppercase ?

Reply:
Whoops, that wasn't supposed to happen. I just fixed it.